.travis/preinstall

```diff
@@ -1,18 +1,32 @@
 #!/bin/bash
 # SPDX-License-Identifier: MIT
 
-set -ex
+# Install package specified by user in LSR_EXTRA_PACKAGES. Executed by Travis
+# during before_install phase.
+#
+# LSR_EXTRA_PACKAGES, set by user in .travis/config.sh, is a space separated
+# list of packages to be installed on the Travis build environment (Ubuntu).
 
-SCRIPTDIR=$(dirname $0)
-CONFIG=${SCRIPTDIR}/config.sh
+set -e
 
-if [[ -f ${CONFIG} ]]; then
-  . ${CONFIG}
+SCRIPTDIR=$(readlink -f $(dirname $0))
+
+# Include library.
+. ${SCRIPTDIR}/utils.sh
+
+# Add python3-selinux package (it turned out that in some circumstances, i.e
+# if selinux policy is enforcing, selinux is required by molecule even if
+# running simple test).
+if lsr_compare_pythons python -eq /usr/bin/python3; then
+  LSR_EXTRA_PACKAGES='python3-selinux'
 fi
 
+# Include config.
```

Comment:
Actually the comments about including config or utils could be removed because the actual code to do this is simple enough that the comments do not explain something new.